requirements.txt

```diff
@@ -1,5 +1,5 @@
-django>=1.11,<2.3
+django>=2.2,<3.3
```

Comment:
django 3.3 ? Do you mean to put django 3.1 here? :smile:

Reply:
@atb00ker is right, please correct

Reply:
Yeah, my mistake